src/Microsoft.Dnx.Runtime/Project.cs

```diff
@@ -93,6 +94,8 @@ public string ProjectDirectory
 
         public ProjectFilesCollection Files { get; private set; }
 
+        public IReadOnlyDictionary<string, PackageFilesSource> PackageFiles { get; private set; }
```

Comment:
Is there any reason to no merge this in with the existing project files collection?